core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -396,6 +397,8 @@ oppia.controller('CreatorDashboard', [
             $scope.commitMessage = description;
             $scope.reviewMessage = null;
             $scope.summaryMessage = null;
+            $scope.isEditButtonShown = true;
```

Comment:
Do not start a boolean variable with `is`, maybe we can make it EditButtonIsVisible or Shown. In genral you can make a boolean variable in a way such that `if + variable name` should make sense, Like `if EditButtonIsVisible` instead if `if  isEditButtonShown`. (Ditto below) 

(We try to only start with is/are for function names, since in general function names start with verbs to help distinguish them from variables.)

Reply:
ok sir